### ISSUE TITLE
Switch to KiCAD or FOSS alternative

### ISSUE THREAD

**Issue by bggardner** [cutiepi-io/cutiepi-board#9]
I love this project, but it's a stretch to claim it's open-source if closed-source software is required to modify the design files.  Please consider switching to KiCAD, etc., or at least post a KiCAD-friendly version of the source.  Thanks!

**Comment by boelle** [comment]
i agree and said the same here: https://github.com/cutiepi-io/cutiepi-board/issues/12

**Comment by penkchen** [comment]
@bggardner @boelle Thank you for the issue and comment, I understand your point. 

When we started the design of current version, we needed a lot of external help to mass produce it. 
From circuit layout, routing, CAD modelling, and PCBA prototyping. All these contractors or companies we worked with were not using KiCAD, as you may have already known, proprietary software is still the de facto in this industry. 

However we learned a lot during the process, and hopefully in the future version of the design, we can introduce FOSS format into some of the process, just like The Raspberry Pi did with CMIO v3 to CMIO v4.
